### DISCUSSION TITLE
精簡詞表+支援異體

### DISCUSSION THREAD

**Discussion by hfhchan** [CanCLID/rime-cantonese-upstream#23] [Ideas]
建議只收其中一款，然後由 rime/rime-cantonese derive 其他寫法同讀音
唔一定要揀一個常用，最緊要係方便 derive，儘量唔好同其他意思重疊，方便寫 rule

- 來: loi4 (而家字表得 loi4，唔收 lai4, lei4)
- 坺: pet6 (而家字表得 `坺`)
- 蝦: haa1 (而家字表得 `蝦`，但係係咪揀過第二個好啲？)
- 砐: ngap6 (ngap6 tau2, ngap6 ngap6 haa5, ngap6 dang3)? 而家字表係岌，但係岌同岌岌可危撞
-----

- 𡃶: `sek3`, derive `錫` `惜`
- 屎胐: `si2 fat1`, derive `屎窟` `屎忽`

-----

- 嚟: `lai4`, derive `lei4` (而家字表得 lai4，唔收 lei4)
- 呢: `ni1`, derive `nei1` (邊個 default 少啲 false positive?)
- 錫: `sek3`, derive `sek6`
- 券: `gyun3`, derive `hyun3`
- 立: `laap6`, derive `lap6` (立亂 lap6 lyun2, 亂立立 lyun6 lap6 lap6 等只可以讀 lap6, 不要 derive laap6)
- 赫 黑 克 刻 剋 尅 氪 勀 etc: `haak1`, derive `hak1` (有冇 case 唔應該 derive？)

-----
另外應該 doc 低邊啲陰聲調都係加 ng，例如「啱 ngaam1」「噏 ngap1」

but 噏耷 ap1 dap1?

@graphemecluster

**Comment by hfhchan** [comment]
仲有個 `囄: lei4`，Unicode 冇香港源，但係 kCheungBauer 有

**Comment by hfhchan** [comment]
Also ban 一啲好明顯嘅錯別字? e.g. `脾` `bei2` 應該係 `髀`, `脾` 留返畀 `pei4` `脾氣` `脾臟`

**Comment by graphemecluster** [comment]
我哋使唔使 normalize 啲 compatibility ideographs？

**Reply by graphemecluster** [reply]
@hfhchan

**Reply by hfhchan** [reply]
仲有咩？輸入法唔應該出 Compat char，要就 SVS/IVS

**Reply by hfhchan** [reply]
可以加 script to verify 冇 compatibility block 嘅字 except a few characters:

> These even include [a few regular ideographs](https://en.wikipedia.org/wiki/CJK_Unified_Ideographs#CJK_Compatibility_Ideographs) (with the Unified_Ideograph property) that do not have duplicates (U+FA0E–U+FA0F, U+FA11, U+FA13–U+FA14, U+FA1F, U+FA21, U+FA23–U+FA24 and U+FA27–U+FA29).[[4]](https://en.wikipedia.org/wiki/CJK_Compatibility_Ideographs#cite_note-4)

Note U+FA11 is now considered unifiable to 崎, but it will not be retroactively turned back into a compatibility character. Also U+FA1F is potentially unifiable to 臈, ditto. U+FA23 is duplicated with U+27EAF.

U+FA20 is incorrectly unified to U+8612.

**Comment by hfhchan** [comment]
而家字表用`嚿`唔用`舊`，但係 essay 得 395，變咗排得勁後。我哋要諗點樣將呢啲冇咩爭議嘅正字泵返高個頻率。
